pkg/apis/cluster/types.go

```diff
@@ -217,7 +197,7 @@ type ResourceModel struct {
 type ResourceModelRange struct {
 	// Name is the name for the resource that you want to categorize.
 	// +required
-	Name ResourceName
+	Name corev1.ResourceName
```

Comment:
I want to confirm one thing, this modification should not introduce any compatibility issues, right?

Reply:
good viewpoint

Reply:
if I deploy karmada by master branch, and then replace the image of aa、controller-manager、scheduler by this commit, if it support extensions, can it verify is compatible？

Reply:
I understand that it is possible. The Cluster object should already exist before the upgrade, and after the upgrade, the Cluster resources can still be used normally, which should meet the requirements for upgrading.

Reply:
I did a carefully test as above described, I is actually compatible~